components/badges.css

```diff
@@ -0,0 +1,64 @@
+.badge-box {
+	position: relative;
+}
+.img-badge {
+	padding: 2px;
+	border-radius: 50%;
+	border: 2px solid white;
+	display: flex;
+	color: white;
+	justify-content: center;
+	align-items: center;
+	position: absolute;
+	font-size: 0.75em;
+	box-sizing: border-box;
+}
+.badge-online {
+	background-color: #00b300;
```

Comment:
Use color variable here instead of hexcode, same for dnd

Reply:
These colors are not supposed to change, so that's why I used the actual hexcode